tests/integration/test_tls.py

```diff
@@ -202,11 +196,8 @@ async def test_disable_tls(ops_test: OpsTest) -> None:
         f"{app}:certificates", f"{TLS_APP_NAME}:certificates"
     )
 
-    # Wait for hooks start reconfiguring app
-    await ops_test.model.block_until(
-        lambda: ops_test.model.applications[app].status != "active", timeout=4 * 60
-    )
-    await ops_test.model.wait_for_idle(apps=[app], status="active", timeout=15 * 60)
+    # Allow time for reconfigure
+    sleep(30)
```

Comment:
Can we test that certificate really changed?
e.g. following command should be able to fetch CA
```
openssl s_client -starttls mysql -connect thehost:3306
```

Reply:
noice, will include that

Reply:
Included on the test app [PR#1](https://github.com/canonical/mysql-test-app/pull/3/)